modules/trusted-firmware-m/Kconfig

```
@@ -20,6 +20,18 @@ config TFM_BL2
 	bool
 	default n
 
+config TFM_SECURE_UART1
+	bool "Use UART1 for TF-M logging"
```

Minor nit: consider remove `Use`:
http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/build/kconfig/tips.html#prompt-strings

> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.

Maybe just write:
```suggestion
	bool "TF-M logging on UART1"
```

---

Done :)